<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoSavingsDialog</class>
 <widget class="QWidget" name="AutoSavingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EcoinShare</string>
  </property>
  <widget class="QLineEdit" name="savingsPercentEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter the percent to share (whole numbers)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter % to share</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="savingsAddressEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter Ecoin address</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>EgAiyRYKKsQEDsbkYuML9d48WUm5DMTdAw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>EcoinShare allows you to automatically send up to 50% of your stake to another Ecoin address after it matures.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="savingsMinEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Do not send less than this (whole numbers)</string>
   </property>
   <property name="placeholderText">
    <string>Enter Min Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="savingsMaxEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Do not send more than this (whole numbers)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Max Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enableButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Check and turn on EcoinShare</string>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disableButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Turn off EcoinShare</string>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addressBookButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>28</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose an address from the address book</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../bitcoin.qrc">
     <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="savingsChangeAddressEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Change address must be owned by you</string>
   </property>
   <property name="placeholderText">
    <string>Enter Change Address (optional)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeAddressBookButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>28</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose an address from the address book</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../bitcoin.qrc">
     <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Donation Percent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min(optional)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max(optional)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Donation Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="freeAddressEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter Ecoin address</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>EgAiyRYKKsQEDsbkYuML9d48WUm5DMTdAw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freeAddressBookButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>28</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose an address from the address book</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../bitcoin.qrc">
     <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="freelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Free Ecoins for running the app each day!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freeRequestButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Get Free Ecoins</string>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="freeMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>EgAiyRYKKsQEDsbkYuML9d48WUm5DMTdAw - Ecoin Bounty Fund</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freeDisableButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Get Free Ecoins</string>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freeReferButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Friend</string>
   </property>
   <property name="icon">
    <iconset resource="../bitcoin.qrc">
     <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
